site/content/docs/manifest/pipeline.md

```diff
@@ -11,6 +11,8 @@ List of all available properties for a Copilot pipeline manifest.
       properties:
         branch: main
         repository: https://github.com/<user>/sample-app-frontend
+        # Optional: specify the name of an existing CodeStar Connections connection.
+        connection_name:
```

Comment:
Should there be an example connection name here?

Reply:
Added. Thanks!